ecs_fargate/README.md

```diff
@@ -870,3 +884,7 @@ Need help? Contact [Datadog support][18].
 [56]: https://app.datadoghq.com/process
 [57]: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#other_task_definition_params
 [58]: https://www.datadoghq.com/blog/monitor-fargate-processes/
+[59]: https://docs.aws.amazon.com/batch/latest/userguide/create-compute-environment.html
+[60]: https://docs.aws.amazon.com/batch/latest/userguide/create-job-queue-fargate.html
+[61]: https://docs.datadoghq.com/resources/json/datadog-agent-aws-batch-ecs-fargate.json
+[62]: /containers/guide/aws-batch-ecs-fargate
```

Comment:
I think this link may be causing the APW README deployment to fail; could you update or investigate this please? 
Putting the `assets/no-deploy` label on this until thats resolved; but once thats fixed feel free to remove the label

Reply:
Yes, this link depends on the prior merging of https://github.com/DataDog/documentation/pull/23187 — must merge that one before this one will work

Reply:
I'm not sure if hard relative links like that work at all? (I could definitely be misremembering)

Reply:
Ah good point, it'll work in the corpsite build but not if someone's just looking at github docs. I'll fix